src/v2/Assets/Theme.ts

```diff
@@ -2,18 +2,18 @@ import { breakpoints } from "@artsy/palette"
 import Colors from "./Colors"
 
 export default {
-  breakpoints: [768, 900, 1024, 1192],
+  breakpoints: ["768px", "900px", "1024px", "1192px"],
```

Comment:
@dzucconi - I don't think this relates to styled-system, we should investigate before making a changge here. Stuff in this assets folder is olddd.

Reply:
Yeah I can pull this off, I thought it may have been handing this off to a theme provider. But the issue turned out to be a discrepancy with the styled grid component